primitive/dune

```diff
@@ -0,0 +1,25 @@
+(library
+ (name primitive)
+ (synopsis "Coq Primitive Objects")
+ (public_name coq-primitive)
+ (wrapped false)
+ (flags (:standard -no-rectypes))
```

Comment:
Is there a better way to disable `rectypes`? My understanding is that `:standard` inherits from the `(env)` in the `dune` file at the root.

Reply:
You can do `:standard \ -rectypes` perhaps?